### ISSUE TITLE
Web - Workspace - Plaid and connect manually option are not displayed in offline

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#11866]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to https://staging.new.expensify.com/
2. Login with Expensifail account
3. Begin the process of adding a bank account but exit in the middle of it
4. Disable the internet connection in the device
5. Click on 'Connect bank account' in workspace settings again and you should see the "Start over" button
6. Verify you are redirected to the Connect bank account screen displaying the options to connect via Plaid or connect manually

## Expected Result:
Screen with options to connect via Plaid or connect manually should be displayed before going online. This is from the TR regression steps https://expensify.testrail.io/index.php?/tests/view/2886612

## Actual Result:
You appear to be offline message is displayed. Options to connect via Plaid or connect manually could be displayed only online.

## Workaround:
unknown
## Platform:

Where is this issue occurring?

- Web

**Version Number:** v1.2.15-3
**Reproducible in staging?:** y
**Reproducible in production?:** could not check
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/195930192-21e82a50-dacb-4cb5-9d3e-eb13784e3677.mp4

**Expensify/Expensify Issue URL:** 
**Issue reported by:** Applause internal team
**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by thienlnam** [comment]
I'm pretty sure this is expected behavior - is there a testrail that indicates this should be possible?

**Comment by thienlnam** [comment]
Yeah, please update the testrail - if you are offline seeing that screen is expected behavior

**Comment by thienlnam** [comment]
@kavimuru Are you able to update this testrail?

**Comment by kbecciv** [comment]
@thienlnam Sent request for updating TR, thank you

**Comment by isagoico** [comment]
@thienlnam Hello! Those steps were extracted from this document https://docs.google.com/document/d/18ew6fHESSoS1FruhNhD6mPEd4p419TkVDXIS3aQQdxs/edit. We will update the steps so the "You appear to be offline" screen is the expected behavior now. 
Let me know if this is not the case and the expected behavior should still be the one described in the document.

**Comment by thienlnam** [comment]
@isagoico Yes the new 'You appear to be offline' screen is the expected behavior now, sorry for the confusion

**Comment by laurenreidexpensify** [comment]
@thienlnam can you confirm if this needs to move to Upwork now? Thanks!

**Comment by thienlnam** [comment]
@laurenreidexpensify We're just waiting for some Applause internal changes here - no action required here yet

**Comment by thienlnam** [comment]
@isagoico Have you had a chance to update the testrail?

**Comment by isagoico** [comment]
Yep! These have been updated already 🙇‍♀️

**Comment by laurenreidexpensify** [comment]
@thienlnam what are next steps? Can I post to Upwork?
